source/_patterns/04-pages/grid-examples/three-column/article-3-col.twig

```diff
@@ -0,0 +1,47 @@
+{% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['grid', 'grid--article']) }) -%}
```

Comment:
The name`grid--article` should be changed as some articles may use a different grid.